src/js/index.js

```diff
@@ -0,0 +1,48 @@
+const capitalize = string => string[0].toUpperCase() + string.substr(1, string.length);
+
+const reverseString = string => string
+  .split('')
+  .reverse()
+  .join('');
+
+const getChar = (char) => {
+  let charCode = char.charCodeAt(0) + 3;
+  if (charCode > 90 && charCode < 97) {
+    charCode = 65 + (charCode - 90) - 1;
+  }
+  if (charCode > 122) {
+    charCode = 97 + (charCode - 122) - 1;
+  }
+  return charCode;
+};
+
+const caesar = (string) => {
+  const newString = string.split('');
+  for (let i = 0; i < newString.length; i += 1) {
+    if (newString[i].match(/\w/)) {
```

Comment:
We want to be able to catch special characters like dot(.) and not change them
![11](https://user-images.githubusercontent.com/37092867/73429857-ee19fa00-433c-11ea-8d1a-cb30156e98c6.png)

**I think you are doing fine with this though, just correct the issue with the key and it should work**